entrypoint.sh

```diff
@@ -11,11 +12,23 @@ if [[ -z "${OWNER}" ]]; then
   exit 2;
 fi
 
+if [[ -n "${WORKING_DIR}" ]]; then
+  echo "Changing to directory: ${WORKING_DIR}"
+  if ! cd "./${WORKING_DIR}"; then
+    echo "Failed to change to directory: ${WORKING_DIR}"
+    exit 1
+  fi
+fi
+
 git config --global --add safe.directory /github/workspace
 
 echo "Verifying the version matches the gem version"
 VERSION_TAG=$(echo $GITHUB_REF | cut -d / -f 3)
 GEM_VERSION=$(ruby -e "require 'rubygems'; gemspec = Dir.entries('.').find { |file| file =~ /.*\.gemspec/ }; spec = Gem::Specification::load(gemspec); puts spec.version")
+# pre release versions have a .pre. prefix after the minor version
+# '1.2.3.pre.beta.4' -> '1.2.3-beta.4'
```

Comment:
_Technically_ you could just set your tag to `1.2.3.pre.beta.4`... but it seems a little strange to alter your tag scheme because of a quirk with bundle lock files.

Especially since typical semvar/Gemfile/version.rb files will all use `1.2.3-beta.4`